packages/components/field-label/src/field-label.tsx

```diff
@@ -81,6 +82,17 @@ export type TFieldLabelProps = {
 };
 
 const FieldLabel = (props: TFieldLabelProps) => {
+  const { theme } = useTheme();
+
+  const getInfoTextTone = (
+    { tone }: Pick<TFieldLabelProps, 'tone'>,
+    theme: 'default' | 'test'
+  ) => {
+    if (theme === 'default') {
+      return tone;
+    } else return 'secondary';
+  };
```

Comment:
```suggestion
import { useTheme, type ThemeName } from '@commercetools-uikit/design-system';
const getInfoTextTone = (theme: ThemeName, tone?: TFieldLabelProps['tone']) => {
  if (theme === 'default') {
    return tone;
  }
  return 'secondary';
};
```
Other than that I don't think we need to define it inside the component.

Reply:
Thank you @kark! 🍫 So you mean I should create a `field-label.styles.tsx` file and move the function there?

Reply:
Cool 🙌 I would only opt for defining the function outside the scope of `const FieldLabel = (props: TFieldLabelProps) => {  }` since it's a pure function, to not define the function on each render of the component. A separate file is not necessary.

Reply:
Sounds good! Thanks!